docs/docs/ref-08-reconciliation.md

```diff
@@ -76,7 +76,7 @@ After the attributes have been updated, we recurse on all the children.
 
 ### Custom Components
 
-We decided that the two custom components are the same. Since components are stateful, we cannot just use the new component and call it a day. React takes all the attributes from the new component and calls `component[Will/Did]ReceiveProps()` on the previous one.
+We decided that the two custom components are the same. Since components are stateful, we cannot just use the new component and call it a day. React takes all the attributes from the new component and calls `component[Will/Did][ReceiveProps/Update]()` on the previous one.
```

Comment:
I think at this point in the cycle it's just `componentWillReceiveProps` - `componentDidUpdate` is called later (the next step after this is render which is mentioned below). `componentWillUpdate` is called before render though and I think that's what this was actually meant to be initially. Let's just use that (and get rid of the brackets entirely, just write out both method names).